logging/google/cloud/logging/logger.py

```diff
@@ -123,19 +128,24 @@ def _make_entry_resource(self, text=None, info=None, message=None,
         :type timestamp: :class:`datetime.datetime`
         :param timestamp: (Optional) timestamp of event being logged.
 
+        :type resource :class:`google.cloud.logging.resource.Resource`
```

Comment:
Can you use a tilde before the `google`:

```
:class:`~google.cloud.logging.resource.Resource`
```

This makes the links look much better (would be `Resource` instead of the full path)